roles/api-environment/templates/express-api/start-express-container.sh.j2

```diff
@@ -40,36 +25,30 @@ if [[ "$(docker ps -aq -f name=$api_name)" ]]; then
     done
 fi
 
-# Remove image if it already exists
-image_id=$(docker images -q $api_name)
-if [[ $image_id ]]; then
-  echo "Remove existing image $image_id"
-  docker rmi $image_id --force
-fi
-
 # Create logs folder and make sure the API have access to wrtie logs into it
 mkdir -p $api_path/logs
 chown -R {{ api_user }}:{{ api_group }} $api_path/logs
 
-# Load docker image from the artifact
-docker load --input $archive
-
 # Run API in a container
 uid="$(id -u {{ api_user }})"
 gid="$(id -g {{ api_user }})"
 docker run -d \
            --user "$uid:$gid" \
            --workdir /usr/src/$api_name \
+           --env API_NAME=$api_name \
+           --env FOREVER_ROOT=/usr/src/$api_name/.forever \
+           --env HOME=/usr/src/$api_name \
            --env-file $env \
            --volume /apis/keytool_files:/usr/src/$api_name/keytool_files \
```

Comment:
Should this keytool files bind mount be read only, similar to here? https://github.com/osu-mist/ansible-roles/blob/9a5c6ed8aa9c9be4f7194f9945893afba93ee19f/roles/api-environment/templates/dropwizard-api/start-container.sh.j2#L66-L67
I think the main concern here would be if one of our express apis were to accidentally write to one of the keytool files, it could cause issues with all other express apis as well as itself